com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -112,7 +112,7 @@ public unsafe void SendClientRpc(ref FastBufferWriter writer, uint rpcMethodId,
             if (sendParams.Send.TargetClientIds != null)
             {
                 // Copy into a localArray because SendMessage doesn't take IEnumerable, only IReadOnlyList
-                ulong* localArray = stackalloc ulong[sendParams.Send.TargetClientIds.Count()];
```

Comment:
RPC params did allocate by using LINQ .Count on IEnumerable. Using IReadonlyList now. This was problematic in the first point because of multiple enumeration.